tests/console/yast2_proxy.pm

```diff
@@ -67,14 +67,19 @@ sub run {
     script_run("yast2 squid; echo yast2-squid-status-\$? > /dev/$serialdev", 0);
 
     # check that squid configuration page shows up
-    assert_screen 'yast2_proxy_squid';
+    assert_screen([qw(yast2_proxy_squid yast2_still_susefirewall2)], 60);
+    if (match_has_tag 'yast2_still_susefirewall2') {
+        record_soft_failure 'bsc#1064405';
+        send_key 'alt-c';
+    }
 
     # enable service start
     send_key_until_needlematch 'yast2_proxy_service_start', 'alt-b';    #Start service when booting
 
     # if firewall is enabled, then send_key alt-p, else move to page http ports
-    if (check_screen 'yast2_proxy_firewall_enabled') { send_key 'alt-p'; }
-    wait_still_screen 3;
+    # disable checking firewall for now, we can enable this again when firewalld works here
+    # if (check_screen 'yast2_proxy_firewall_enabled') { send_key 'alt-p'; }
+    # wait_still_screen 3;
```

Comment:
Don't we need this for TW?

Reply:
at moment no.

Reply:
Let's either remove this code completely, or trigger this code if it's not affected by bsc#1064405. Otherwise looks fine.